lang/en/tool_mergeusers.php

```diff
@@ -186,8 +186,15 @@
     data from new user (by default). This also means that data from old user is 
     deleted to keep the consistence. Otherwise, if you uncheck this option, 
     data from old user will be kept.';
-
 $string['starttime'] = 'Started merging at {$a}';
 $string['finishtime'] = 'Finished merging at {$a}';
 $string['timetaken'] = 'Merge took {$a} seconds';
 $string['privacy:metadata'] = 'The Merge User Accounts plugin does not store any personal data.';
+$string['queue_to_process_pending_merge_requests'] = 'Get queue of the pending merge requests and launch adhoc task for each single request';
+$string['cannotfinduser'] = 'Cannot find the user. Change the input parameters.';
```

Comment:
here is the place where you should use the variable `$a` to show the parameters you added before. See core examples, but the idea is something like: `$string['cannotfinduser'] = 'ERROR: Cannot find user with {$a->userfield} {$a->uservalue}.';`